app_dart/lib/src/datastore/cocoon_config.dart

```diff
@@ -75,6 +75,10 @@ class Config {
     return await getBuilders(Providers.freshHttpClient, loggingService, twoSecondLinearBackoff, bucket);
   }
 
+  Future<List<Map<String, dynamic>>> getRepoLuciBuilders(String bucket, String repo) async {
+    return await getRepoBuilders(Providers.freshHttpClient, loggingService, twoSecondLinearBackoff, bucket, repo);
```

Comment:
Nit: it seems a bit over complicated to wrap the function from utils.dart here.

Reply:
Have simplified `getRepoBuilders` by removing the error catch logic. Similar to `getBranches`, it makes calling/testing the logic clear and simple by putting it in utils.